client/jetpack-onboarding/steps/site-title.jsx

```diff
@@ -37,11 +38,12 @@ class JetpackOnboardingSiteTitleStep extends React.PureComponent {
 		this.setState( { title: event.target.value } );
 	};
 
-	submit = () => {
+	handleSubmit = () => {
 		this.props.saveJetpackOnboardingSettings( this.props.siteId, {
 			siteTitle: this.state.title,
 			siteDescription: this.state.description,
 		} );
+		page( this.props.getForwardUrl() );
```

Comment:
Did you mean `page.redirect()` instead of `page()`?

Reply:
I believe they are equivalent. Happy to use `page.redirect()` as it is more indicative of what it actually does.